### ISSUE TITLE
Crash with Epic Fight Mod 

### ISSUE THREAD

**Issue by rainysux** [Asek3/Oculus#200]
### Minecraft Version

Minecraft 1.16.5 Forge

### Iris Version

forge

### Sodium Version

forge

### Operating System

Windows 11

### What is your GPU?

Ryzen 7 3700X

### Java Version

Java 8

### Reproduction Steps

Walk around with Epic Fight + Oculus 
Eventually Crashes when switching through modes

### Crash Report file and latest.log

```shell
https://gist.github.com/rainysux/ccd5c7ffa495e1c6241dc0d170ef0ba1
```

### Additional context

It only crashes when running Oculus shaders and when I play without shaders, it works normally without problems.  (I am sorry for the big mod pack). This has happened with other mod packs as well with these two mods together.

**Comment by dreckz** [comment]
Bumping this same Crash.  Experiencing this as well.

Minecraft Version
Minecraft 1.16.5 Forge

Forge Version
36.2.39

Oculus Version
1.2.6

Rubidium Version
0.2.10

Operating System
Windows 11

What is your GPU?
RTX 3060

Java Version
Java 8

Reproduction Steps
Walk around with Epic Fight + Oculus where a spawning is spawning a riding entity

Crash Report file and latest.log
LATEST:
[latest.log](https://github.com/Asek3/Oculus/files/10035523/latest.log)
CRASH:
[crash-2022-11-17_14.43.10-client.txt](https://github.com/Asek3/Oculus/files/10035525/crash-2022-11-17_14.43.10-client.txt)

Additional context
Occurs when shaders are on, but not when turned off.

I also confirmed this is a client side issue as I had tested this crash on a multiplayer server and the server did not crash.
